CHANGELOG.rst

```diff
@@ -31,7 +50,7 @@ v5.0.0
 Release Summary
 ---------------
 
-Starting from this release, the minimum `ansible-core` version this collection requires is `2.14.0`. The last known version compatible with ansible-core<2.14 is `v4.0.3`.
+Starting from this release, the minimum `ansible-core` version this collection requires is `2.14.0`. The last known version compatible with
```

Comment:
This looks broken?

Reply:
no no, someone manually updated it, and the tool put's it in to the correct place.

Reply:
Well, the sentence is incomplete here. Need to add words after: "The last known version compatible with".

Reply:
fixed